sdk/spring/spring-cloud-azure-integration-tests/src/test/java/com/azure/spring/cloud/integration/tests/eventhubs/binder/EventHubsBinderManualModeIT.java

```diff
@@ -65,6 +66,11 @@ Consumer<Message<String>> consume() {
                 }
             };
         }
+
+        @ServiceActivator(inputChannel = "errorChannel")
+        public void processError(Message sendFailedMsg) {
+            LOGGER.info("receive produce error message: '{}'", sendFailedMsg.getPayload());
```

Comment:
nit: Should we also update the info, from `receive produce error message` to `receive error message`?